### DISCUSSION TITLE
how to reload Definitions via GraphQL

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18382] [Q&A] [answered]
If in my Dagster code I'm using `Definitions` instead of `repository`, will code specified in [the docs](https://docs.dagster.io/concepts/webserver/graphql-client#reloading-all-repositories-in-a-repository-location) work for reloading my code locations? More specifically, I mean this GraphQL code:
```python
from dagster_graphql import ReloadRepositoryLocationInfo

reload_info: ReloadRepositoryLocationInfo = client.reload_repository_location(REPO_NAME)
```

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1701261928616819?thread_ts=1701261928.616819&cid=C01U954MEER).*

**Comment by smackesey** [comment, marked as answer]
Yes, it will work. Unfortunately the GQL client still uses some old API names, it is in our queue to fix that.

**Reply by svagier** [reply]
Thank you @smackesey  for the answer, appreciate it!

**Comment by dridk** [comment]
Hi, 
It doesn't work. I have to refresh de webrowser with F5 to make it work.
it would be helpfull to reload the definition in the UI using a command line.   

```python

from dagster_graphql import (
    ReloadRepositoryLocationInfo,
    ReloadRepositoryLocationStatus,
    DagsterGraphQLClient,
)

client = DagsterGraphQLClient("localhost", port_number=3000)
reload_info: ReloadRepositoryLocationInfo = client.reload_repository_location("vision")
if reload_info.status == ReloadRepositoryLocationStatus.SUCCESS:
    print("reload")
else:
    raise Exception(
        "Repository location reload failed because of a "
        f"{reload_info.failure_type} error: {reload_info.message}"
    )

```

**Comment by smackesey** [comment]
@dridk Thanks for the report, I can confirm it's no longer working and have filed this as a bug.
